athenet/sparsifying/utils/interval.py

```diff
@@ -0,0 +1,364 @@
+"""Intervals in Theano including special functions for sparsifying."""
+
+from athenet.sparsifying.utils.numlike import Numlike
+from theano import function
+from theano import tensor as T
+from theano.ifelse import ifelse
+
+
+class Interval(Numlike):
+    """Theano interval matrix class
+
+    represents matrix of interval. Behaves like limited numpy.ndarray.
```

Comment:
interval -> intervals :)

"Behaves like limited numpy.ndarray of intervals" maybe?